lib/rules/web/admin_console/4.2/monitoring_alerts.xyaml

```diff
@@ -195,8 +195,23 @@ check_view_metrics:
     text: View in Metrics
     link_url: /monitoring/query-browser?query0=vector(1)
   action: check_link_and_text
+click_view_metrics:
+  params:
+    text: View in Metrics
+    link_url: /monitoring/query-browser?query0=vector(1)
+  action: click_link_with_text
 check_alert_description_on_rule_detail:
   params:
     text: This is an alert meant to ensure that the entire alerting pipeline is functional
     link_url: /monitoring/alerts
   action: check_link_and_text
+
+hide_alert_graph:
+  action: hide_show_alert_graph_button
+show_alert_graph:
```

Comment:
hide_alert_graph and show_alert_graph call for the same function hide_show_alert_graph_button
I don't think it is needed, but it makes sense that user can know the function by the name

Reply:
I suppose this way is better and make code in feature file is easy to read.